lib/signaling/v2/room.js

```diff
@@ -665,12 +665,21 @@ function periodicallyPublishStats(roomV2, transport, intervalMs) {
         // StatsReport will add zeros to properties that do not exist.
         const report = new StatsReport(id, response, true /* prepareForInsights */);
 
-        transport.publishEvent('quality', 'stats-report', {
-          audioTrackStats: report.remoteAudioTrackStats,
-          localAudioTrackStats: report.localAudioTrackStats,
-          localVideoTrackStats: report.localVideoTrackStats,
-          peerConnectionId: report.peerConnectionId,
-          videoTrackStats: report.remoteVideoTrackStats
+        transport.eventObserver.emit('event', {
+          group: 'quality',
+          level: 'info',
+          name: 'stats-report',
+          payload: {
+            audioTrackStats: report.remoteAudioTrackStats.map((trackStat, i) =>
+              addAVSyncMetricsToRemoteTrackStats(trackStat, response.remoteAudioTrackStats[i])),
```

Comment:
@manjeshbhargav, instead of adding the AVSyncMetric after the report is generated, Why not add this in `StatsReport` itself so that you don't have to patch it later?

Reply:
@makarandp0 I don't want these stats to be part of the public StatsReport class. I propose that we add them in a separate release.

Reply:
I wasn't expecting the public APIs to change as a part of this ticket so patching the events reported to insights is 👌. Maybe it deserves a comment explaining why the stats are added later.